asset/quickstart/get-saved-query/getSavedQuery.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2022 Google LLC
```

Comment:
Code file names should be snake_case, e.g. "get_saved_query.go".

Reply:
changed in the new commit:)